### DISCUSSION TITLE
Calculating COM for coarse-grained structure

### DISCUSSION THREAD

**Discussion by srdjan125** [plumed/plumed2#886] [Q&A] [answered]
Hello,

I am trying to analyze my trajectory from coarse-grained MD trajectory (done by gromacs) and part of my plumed-wham.dat file has following commands:

Dim1: GROUP NDX_FILE=index.ndx NDX_GROUP=DimerA
Dim2: GROUP NDX_FILE=index.ndx NDX_GROUP=DimerB
com1: COM ATOMS=Dim1
com2: COM ATOMS=Dim2
r: DISTANCE ATOMS=com2,com1

I have tried to run the following command:

**_plumed driver --mf_xtc concatenated.xtc --plumed plumed-wham.dat_**

However, when I try to run the script I get an error that it cannot compute COM because masses are unknown (obviously because it is coarse grained structure so plumed cannot recognize atom names and their masses).
When I replace it by CENTER (instead of COM) it works, however I want to calculate COM (as during my GROMACS simulations collective variable was distance between COMs).

Is there a way to provide topology info to plumed (for example to provide gromacs tpr file or something).
Thank you in advance.

**Comment by GiovanniBussi** [comment, marked as answer]
Please check the manual page of [driver](https://www.plumed.org/doc-v2.8/user-doc/html/driver.html).

You can use a special command to dump mass and charges with a short MD ([DUMPMASSCHARGE](https://www.plumed.org/doc-v2.8/user-doc/html/_d_u_m_p_m_a_s_s_c_h_a_r_g_e.html)) and then read it back in driver (with option `--mc`).

**Reply by srdjan125** [reply]
Thank you for answer it worked.
